domino_data/data_sources.py

```diff
@@ -476,6 +476,8 @@ def upload_fileobj(self, object_key: str, fileobj: Any) -> None:
     DatasourceDtoDataSourceType.S3CONFIG: ObjectStoreDatasource,
     DatasourceDtoDataSourceType.SNOWFLAKECONFIG: QueryDatasource,
     DatasourceDtoDataSourceType.SQLSERVERCONFIG: QueryDatasource,
+    DatasourceDtoDataSourceType.TERADATACONFIG: QueryDatasource,
+    DatasourceDtoDataSourceType.TRINOCONFIG: QueryDatasource,  # necessary?
```

Comment:
yeah but please regenerate after https://github.com/dominodatalab/domino-data/pull/52 is landed